server/migrations/20200331142748-create-users.js

```diff
@@ -42,13 +42,16 @@ export function up(queryInterface, Sequelize) {
     facebookId: {
       type: Sequelize.INTEGER,
     },
+    token:{
```

Comment:
Does this mean that you store a token in the DB ?

Reply:
Yes so that we can logout easily

Reply:
But the one who did login, did not store it in DB and I think it is not needed since we are building stateless API

Reply:
Going forward I think this will be done By @benshidanny11  to do the `logout` but this way was the way we agreed upon with @fordarnold and @bdushimi

Reply:
Moreover, I had to make the environment that I will be using, since my work depends on others to finish.